### PR TITLE
Migrate TC test_wipe_out_directory_permissions.py

### PR DESCRIPTION
**Pull request by Sheetalpamecha**
Migrate TC [test_wipe_out_directory_permissions.py](https://github.com/gluster/glusto-tests/blob/master/tests/functional/dht/test_wipe_out_directory_permissions.py)

Updates: #292 
Signed-off-by: Sheetal Pamecha <spamecha@redhat.com>

### CODE REVIEW

**Comment by Sheetalpamecha** [comment]
```
Results for dht
Test nature : disruptive
TestName : test_wipe_out_directory_permissions
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     dist    |     PASS    |        00:01:54       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   1   |
|            Tests Skipped             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 03 minutes 56 seconds

```

File: tests/functional/dht/test_wipe_out_directory_permissions.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    def _check_permissions_of_dir(self):
+        """Check permissions of dir created."""
+        bricks = self.redant.get_all_bricks(self.vol_name,
+                                            self.server_list[0])
+
+        for brick_path in bricks:
+            node, path = brick_path.split(":")
+            ret = self.redant.get_file_stat(node, "{}/dir".format(path))
+            if not ret:
+                raise Exception("Not able to get stat of the dir")
```
Comment (nik-redhat) [inline tests/functional/dht/test_wipe_out_directory_permissions.py:52]: We don't seem to be validating the expected permissions here.

**Review by nik-redhat — CHANGES_REQUESTED** [review]

File: tests/functional/dht/test_wipe_out_directory_permissions.py (hunk 1 above)
Comment (Sheetalpamecha) [reply tests/functional/dht/test_wipe_out_directory_permissions.py:52]: Added the check. But there seems to be a bug here. Directory permissions changes on cd/ls operation on mountpoint without explicit changing from us. Tried it manually too and ran into this bug.

Comment (nik-redhat) [reply tests/functional/dht/test_wipe_out_directory_permissions.py:52]: Ok, so did they have some other permission?

Comment (Sheetalpamecha) [reply tests/functional/dht/test_wipe_out_directory_permissions.py:52]: No

Comment (nik-redhat) [reply tests/functional/dht/test_wipe_out_directory_permissions.py:52]: Ok, it seems to be working on downstream code but not on upstream latest code.
Then, I think it is better to mark it as excluded with this justification.

Comment (Sheetalpamecha) [reply tests/functional/dht/test_wipe_out_directory_permissions.py:52]: okay

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**